repose-aggregator/components/filters/keystone-v2-filter/src/main/scala/org/openrepose/filters/keystonev2/KeystoneRequestHandler.scala

```diff
@@ -187,7 +188,7 @@ class KeystoneRequestHandler(identityServiceUri: String, akkaServiceClient: Akka
     handleResponse("endpoints", akkaResponse, extractEndpointInfo)
   }
 
-  final def getGroups(authenticatingToken: String, forToken: String, checkCache: Boolean = true): Try[Vector[String]] = {
+  final def getGroups(authenticatingToken: String, forToken: String, applyRcnRoles: Boolean, checkCache: Boolean = true): Try[Vector[String]] = {
```

Comment:
This endpoint won't have the `apply_rcn_roles` query param.

Here's the documented Bob-thoughts:
https://rackspace.slack.com/archives/C0DCZSZQX/p1494423428408615